cmd/release-controller/info.go

```diff
@@ -52,8 +58,19 @@ func (c *CachingReleaseInfo) ChangeLog(from, to string) (string, error) {
 	return s, err
 }
 
+func (c *CachingReleaseInfo) GetReleaseInfo(image string) (string, error) {
+	if strings.Contains(image, "\x00") {
```

Comment:
This is not necessary and should be removed.  Someone using this method wrong is just using this method wrong.

Reply:
Still need to remove this if block.